### PR TITLE
admin: add mylibrary and translation mechanism

### PR DESCRIPTION
**Pull request by Garfield-fr**
* add mylibrary components
* add some elements for translation functionality
* change some license files

Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
     "@angular/router": "~8.1.2",
     "@cospired/i18n-iso-languages": "^2.0.5",
     "@ngx-translate/core": "^11.0.1",
-    "@rero/ng-core": "0.0.6",
+    "@rero/ng-core": "0.0.7",
```
Comment (AoNoOokami) [inline package.json]: version 0.0.8 of ng-core has been published

File: projects/admin/src/app/record/custom-editor/circulation-settings/circulation-policy/circulation-policy.component.scss [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
+
```
Comment (sebastiendeleze) [inline projects/admin/src/app/record/custom-editor/circulation-settings/circulation-policy/circulation-policy.component.scss]: as there's no css to the scope of this component, this file should be removed.

File: projects/admin/src/app/record/custom-editor/libraries/exception-dates-edit/exception-dates-edit.component.scss [hunk 3]
```diff
@@ -1,18 +1,16 @@
...
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
```
Comment (sebastiendeleze) [inline projects/admin/src/app/record/custom-editor/libraries/exception-dates-edit/exception-dates-edit.component.scss]: as there's no css to the scope of this component, this file should be removed.

File: projects/admin/src/app/record/custom-editor/libraries/exception-dates-list/exception-dates-list.component.scss [hunk 4]
```diff
@@ -1,18 +1,16 @@
...
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
```
Comment (sebastiendeleze) [inline projects/admin/src/app/record/custom-editor/libraries/exception-dates-list/exception-dates-list.component.scss]: same here

File: projects/admin/src/app/record/custom-editor/libraries/library.component.scss [hunk 5]
```diff
@@ -1,18 +1,16 @@
...
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
```
Comment (sebastiendeleze) [inline projects/admin/src/app/record/custom-editor/libraries/library.component.scss]: same here.

File: projects/admin/src/app/app.component.ts [hunk 6]
```diff
@@ -14,28 +14,50 @@
...
+      }
+      const languages: Array<string> = this.appSettings.languages;
+      this.translate.addLangs(languages);
+      this.translate.setDefaultLang(this.appSettings.defaultLanguage);
+      this.translate.use(data.settings.language);
+      this.appSettings.setSettings(data.settings);
       const currentUser = new User(data.metadata);
-      console.log(currentUser, 'user', data);
-      userService.setCurrentUser(currentUser);
+      this.userService.setCurrentUser(currentUser);
```
Comment (sebastiendeleze) [inline projects/admin/src/app/app.component.ts]: logic for loading and storing the user should be inside user.service.

File: projects/admin/src/app/record/custom-editor/libraries/library.component.scss (hunk 5 above)
Comment (Garfield-fr) [reply projects/admin/src/app/record/custom-editor/libraries/library.component.scss]: done

File: projects/admin/src/app/record/custom-editor/circulation-settings/circulation-policy/circulation-policy.component.scss (hunk 2 above)
Comment (Garfield-fr) [reply projects/admin/src/app/record/custom-editor/circulation-settings/circulation-policy/circulation-policy.component.scss]: done

File: package.json (hunk 1 above)
Comment (Garfield-fr) [reply package.json]: done

File: projects/admin/src/app/app.component.ts (hunk 6 above)
Comment (Garfield-fr) [reply projects/admin/src/app/app.component.ts]: done

**Pull request merged**